defaults/main.yml

```diff
@@ -86,3 +89,24 @@ openwisp2_celery_worker_prefetch_multiplier: 1
 openwisp2_celery_task_acks_late: True
 openwisp2_django_celery_logging: False
 postfix_smtpd_relay_restrictions_override: "permit_sasl_authenticated, permit_mynetworks, check_relay_domains, reject_unauth_destination, reject"
+freeradius_dir: /etc/freeradius/3.0
+freeradius_mods_available_dir: "{{ freeradius_dir }}/mods-available"
+freeradius_mods_enabled_dir: "{{ freeradius_dir }}/mods-enabled"
+freeradius_sites_available_dir: "{{ freeradius_dir }}/sites-available"
+freeradius_sites_enabled_dir: "{{ freeradius_dir }}/sites-enabled"
+freeradius_sql:
+    driver: rlm_sql_postgresql
+    dialect: postgresql
+    host: localhost
+    port: 5432
+    dbname: freeradius
+    user: admin
+    password: admin
```

Comment:
the default installation uses sqlite

Reply:
Yes, I have to work to work with SQLite, In fact I wanted to ask, is SQLite practical? Do people want that? Asking because sometimes I have noticed it has different behavior than PostgreSQL, while it makes sense for develop but just wondering if people need it in production as well!

Reply:
Postgresql is better but the initial assumption is that ansible-openwisp2 would work as a base for people to get started and then they'd configure it as they preferred. Most people will use it with the default setting (sqlite), which will work for most cases.
Advanced cases will require some work on their side to set up either MySQL or PostgreSQL, depending on which one they prefer, I think this is ok for now, especially if we consider that in the near future we'll be migrating to docker, which if I'm not mistaken uses PG by default, right?

Reply:
Yes, Got it.
docker-openwisp uses Postgresql by default.